src/material-experimental/mdc-slider/slider-thumb.ts

```diff
@@ -47,18 +47,36 @@ export interface MatSliderDragEvent {
 }) export class MatSliderThumb {
   /** The current value of this slider input. */
   @Input()
-  get value(): number { return this._value; }
-  set value(v: number) { this._value = v; }
-  private _value: number;
+  get value(): number { return Number(this._elementRef.nativeElement.getAttribute('value')); }
+  set value(v: number) {
+    this._initialized = true;
+
+    // If the foundation has already been initialized, we need to
+    // relay any value updates to it so that it can update the UI.
+    if (this._slider._initialized) {
+      this._slider._setValue(v, this.thumb);
+    } else {
+      // Setup for the MDC foundation.
+      this._elementRef.nativeElement.setAttribute('value', v.toString());
+    }
+  }
 
   /** The minimum value that this slider input can have. */
   @Input()
-  get min(): number { return 0; }
+  get min(): number {
+    return (this._slider._isRange() && this.thumb === Thumb.END)
+      ? this._slider._getValue(Thumb.START)
+      : this._slider.min;
+  }
```

Comment:
You can define a getter without defining a setter, which makes a readonly property, so you don't need to have a setter that just throws an error. However, our library practice is to avoid using getters unless they're required as an input. Since this property _can't_ be set at all, it shouldn't be an input, meaning it also shouldn't be a getter, and should just be a function. 

Going further, though- why have an API to get the min/max at all on the thumb rather than reading it directly from the parent slider?